lib/plottr_components/src/components/timeline/BlankCard.js

```diff
@@ -54,11 +54,15 @@ const BlankCardConnector = (connector) => {
 
       const json = e.dataTransfer.getData('text/json')
       const droppedData = JSON.parse(json)
-      if (!droppedData.cardId) return
 
       const { beatId, lineId } = this.props
+      if (droppedData.cardIds) {
```

Comment:
Is this the only other place that the user can drop cards?
Thinking about it, it makes sense that this would be the only place, but it's probably best to be sure.